pytorch_lightning/plugins/training_type/ddp_spawn.py

```diff
@@ -416,6 +415,6 @@ def empty(self) -> bool:
 
 class _SpawnOutput(NamedTuple):
     best_model_path: Optional[_PATH]
-    last_path: Optional[_PATH]
+    weights_path: Optional[_PATH]
```

Comment:
You should update the subclasses too here, as their types are not correct now

https://github.com/PyTorchLightning/pytorch-lightning/pull/10034#discussion_r760982179

Reply:
ok boss, updated! ❤️